modules/wolfgame.py

```diff
@@ -787,6 +798,249 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", "pingpref", "ping-if", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference. Usage: 'pingif <players> [once|ping|always]'"""
+    altpinged, players = is_user_altpinged(nick)
+    rest = rest.split()
+    if nick in var.USERS:
+        cloak = var.USERS[nick]["cloak"]
+        acc = var.USERS[nick]["account"]
+    elif is_fake_nick(nick):
+        return # just ignore it
+    else:
+        if chan == nick:
+            pm(cli, nick, "You need to be in {0} to use that command.".format(botconfig.CHANNEL))
+        else: # former message: "You won the lottery! This is a bug though, so report it to the admins."
+            cli.notice(nick, "You need to be in {0} to use that command.".format(botconfig.CHANNEL))
+        return
+
+    msg = []
+    pref_mean = {"once": "pinged immediately",
+                 "ping": "added automatically to the {0}ping list",
+                 "all" : "pinged immediately and added to the {0}ping list"}
+
+    if (not acc or acc == "*") and var.ACCOUNTS_ONLY:
+        msg.append("You are not logged in to NickServ.")
+
+    elif not rest:
+        if altpinged:
+            msg.append("Your ping preferences are currently set to {0}.".format(players))
+            if acc and acc != "*" and acc in var.PING_PREFS_ACCS:
+                msg.append("When your desired player count is reached, you will be {0}.".format(pref_mean[var.PING_PREFS_ACCS[acc]]))
+            elif not var.ACCOUNTS_ONLY and cloak in var.PING_PREFS:
+                msg.append("When your desired player count is reached, you will be {0}.".format(pref_mean[var.PING_PREFS[cloak]]))
+        else:
+            msg.append("You do not have any ping preferences currently set.")
+
+    elif (rest[0].isdigit() and int(rest[0]) == 0) or (len(rest) > 1 and rest[1].isdigit() and int(rest[1]) == 0):
+        if altpinged:
+            msg.append("Your ping preferences have been removed (was {0}).".format(players))
+            toggle_altpinged_status(nick, 0, players)
+        else:
+            msg.append("You do not have any preferences set.")
+
+    elif rest[0].isdigit() or (len(rest) > 1 and rest[1].isdigit()):
+        if rest[0].isdigit():
+            num = int(rest[0])
+        else:
+            num = int(rest[1])
+        if num > 999:
+            msg.append("Please select a number between 0 and {0}.".format(var.MAX_PLAYERS))
```

Comment:
999 is a weird number, instead of printing 999 in the message, maybe should instead just say "That number was too large" (could even raise the limit to 2**31 like stasis, dunno)